src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -1,10 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
     <DefineConstants>$(DefineConstants);RESOURCES_EXTENSIONS</DefineConstants>
+    <!-- https://github.com/dotnet/runtime/issues/39292 tracks finding an alternative to BinaryFormatter. -->
```

Comment:
Nit: I think pragma disable looked better for this one. It is just a single place and it is not ifdef.

Reply:
I agree...